slices/libfreetype6.yaml

```diff
@@ -0,0 +1,12 @@
+package: libfreetype6
+
+slices:
+  libs:
+    essential:
+      - libbrotli1_libs
+      - libbz2-1.0_libs
+      - libc6_libs
+      - libpng16-16t64_libs
+      - zlib1g_libs
+    contents:
+      /usr/lib/*-linux-*/libfreetype.so.6*:
```

Comment:
Please include the copyright slice and its reference similary to what has been suggested in fontconfig-config slice.